src/string/i18n.c

```diff
@@ -12,11 +12,11 @@ char *cr_translate_assert_msg(int msg_index, ...) {
     static char *messages[] = {
         [CRITERION_ASSERT_MSG_FAIL] = N_("The conditions for this assertion were not met."),
         [CRITERION_ASSERT_MSG_EXPR_FALSE] = N_("The expression %s is false."),
-        [CRITERION_ASSERT_MSG_EXPR_AS_STRINGS_FALSE] = N_("The expression (as strings) %s is false."),
+        [CRITERION_ASSERT_MSG_EXPR_AS_STRINGS_FALSE] = N_("The expression (as strings) %s is false: actual=`%s` expected=`%s`."),
         [CRITERION_ASSERT_MSG_IS_NULL] = N_("%s is null."),
         [CRITERION_ASSERT_MSG_IS_NOT_NULL] = N_("%s is not null."),
-        [CRITERION_ASSERT_MSG_IS_EMPTY] = N_("%s is empty."),
-        [CRITERION_ASSERT_MSG_IS_NOT_EMPTY] = N_("%s is not empty."),
+        [CRITERION_ASSERT_MSG_IS_EMPTY] = N_("%s (`%s`) is empty."),
```

Comment:
The second specifier in this message is useless -- we know it's going to be "(\`\`)" since the message is empty.